modules/end-user-portal/src/Pages/Search/Results.cshtml.cs

```diff
@@ -2,18 +2,22 @@
 using GpConnect.NationalDataSharingPortal.EndUserPortal.Core.HttpClientServices.Interfaces;
 using GpConnect.NationalDataSharingPortal.EndUserPortal.Helpers.Enumerations;
 using GpConnect.NationalDataSharingPortal.EndUserPortal.Models;
+using GpConnect.NationalDataSharingPortal.EndUserPortal.Models.Config;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Options;
 
 namespace GpConnect.NationalDataSharingPortal.EndUserPortal.Pages.Search;
 
 public partial class ResultsModel : BaseModel
 {
+    private const int FIRST_PAGE = 1;
```

Comment:
Shouldn't this go into a pageconstants helper class?

Reply:
Extracted